### PR TITLE
feat(donate-section-one): creates and styles component

### PR DESCRIPTION
**Pull request by mcheng09**
# Description

Built out the Donate Page's Section One. 

Screen Shot 2020-09-02 at 10 09 14 PM
Screen Shot 2020-09-02 at 10 09 37 PM

# Links

Design:

https://www.figma.com/file/fUItcdy9fKRHPggnncr2zl/HUOA-Team?node-id=4410%3A156

### CODE REVIEW

File: src/sections/DonateSectionOne/DonateSectionOne.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+
+const DonateSectionOne = ({ data }) => {
+  const columnArray = Object.keys(data.body).map(i => data.body[i]);
+  const cols = columnArray.map((col, i) => {
+    const icons = [
+      <TreePlanting className='donate-section-one__icon'/>, 
+      <CoinInHand className='donate-section-one__icon'/>, 
+      <Car className='donate-section-one__icon'/>
+    ]
```
Comment (mattlaguardia) [inline src/sections/DonateSectionOne/DonateSectionOne.js:17]: Are these selectable in WP?

Comment (mcheng09) [reply src/sections/DonateSectionOne/DonateSectionOne.js]: They are now! I created selectable fields for the ACF

File: .stylelintignore [hunk 2]
```diff
@@ -1,2 +1 @@
 src/styles/normalize.scss
```
Comment (mattlaguardia) [inline .stylelintignore:1]: Why is this getting deleted?

File: src/sections/DonateSectionOne/DonateSectionOne.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
+@import 'styles/fonts';
+@import 'styles/utilities/colors';
+@import 'styles/utilities/breakpoints';
+
+.donate-section-one {
+  margin: 64px auto 32px;
+  
+  &__title {
+    margin: 0 auto 48px;;
```
Comment (mattlaguardia) [inline src/sections/DonateSectionOne/DonateSectionOne.scss:9]:
```suggestion
    margin: 0 auto 48px;
```

File: stylelint.config.js [hunk 4]
```diff
@@ -1,7 +1,11 @@
-// BEM Selector Class Configs
-const customBemSelector = () => {
+// Transform from PascalCase to kebab-case
+const pascalToKebabCase = str => str.replace(/\.?([A-Z])/g, (char) =>  "-" + char.toLowerCase()).replace(/^-/, "");
```
Comment (mattlaguardia) [inline stylelint.config.js:2]: Is this supposed to be in here?

File: src/sections/DonateSectionOne/DonateSectionOne.scss (hunk 3 above)
Comment (mcheng09) [reply src/sections/DonateSectionOne/DonateSectionOne.scss]: Fixed, thanks for catching!

File: stylelint.config.js (hunk 4 above)
Comment (mcheng09) [reply stylelint.config.js]: Nope, I fixed the commit history so this shouldn't be in this PR anymore.

File: .stylelintignore (hunk 2 above)
Comment (mcheng09) [reply .stylelintignore]: I fixed the commit history so this shouldn't be in this PR anymore.

**Review by mattlaguardia — APPROVED** [review]

**Pull request merged**